### ISSUE TITLE
[Facebook] "Cannot parse data" when trying to download Facebook reel

### ISSUE THREAD

**Issue by ivo-ns** [yt-dlp/yt-dlp#7626]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Bulgaria

### Provide a description that is worded well enough to be understood

I'm trying to download https://www.facebook.com/reel/306581085124522 but it's not working.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--cookies-from-browser', 'chrome', 'https://www.facebook.com/reel/306581085124522', '-vU']
[debug] User config "C:\Users\johnl\AppData\Roaming\yt-dlp\config.txt": ['-f', 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best', '-i', '-o', 'D:/Courses/%(title)s.%(ext)s', '--no-mtime']
[debug] Encodings: locale cp1251, fs utf-8, pref cp1251, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2021-02-02-git-2367affc2c-full_build-www.gyan.dev, ffprobe 2021-02-02-git-2367affc2c-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\johnl\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\johnl\AppData\Local\Google\Chrome\User Data\Local State"
[Cookies] Extracted 5469 cookies from chrome
[debug] cookie version breakdown: {'v10': 5477, 'other': 0, 'unencrypted': 0}
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[facebook:reel] Extracting URL: https://www.facebook.com/reel/306581085124522
[facebook] Extracting URL: https://m.facebook.com/watch/?v=306581085124522&_rdr
[facebook] 306581085124522: Downloading webpage
[facebook] 306581085124522: Downloading webpage
ERROR: [facebook] 306581085124522: Cannot parse data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 710, in extract
  File "yt_dlp\extractor\facebook.py", line 707, in _real_extract
  File "yt_dlp\extractor\facebook.py", line 654, in _extract_from_url
```

**Comment by bashonly** [comment]
Duplicate of #4311 (and related: #7469)

to download the reel:
1. run `yt-dlp --update-to nightly`
2. do NOT pass cookies (see discussion in #4311)

&nbsp;
```shell
$ yt-dlp -F "https://www.facebook.com/reel/306581085124522"
[facebook:reel] Extracting URL: https://www.facebook.com/reel/306581085124522
[facebook] Extracting URL: https://m.facebook.com/watch/?v=306581085124522&_rdr
[facebook] 306581085124522: Downloading webpage
[facebook] 306581085124522: Downloading MPD manifest
[info] Available formats for 306581085124522:
ID                  EXT RESOLUTION │   FILESIZE   TBR PROTO │ VCODEC          VBR ACODEC     ABR ASR MORE INFO
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
987017479175882a    m4a audio only │ ~709.62KiB   65k https │ audio only          mp4a.40.5  65k 44k DASH audio, m4a_dash
291290143559503a    m4a audio only │ ≈535.67KiB   49k https │ audio only          mp4a.40.5  49k 44k DASH audio, m4a_dash
sd                  mp4 unknown    │                  https │ unknown             unknown
hd                  mp4 unknown    │                  https │ unknown             unknown
1727177101064097v-0 mp4 216x384    │ ~  1.20MiB  113k https │ vp09.00.20.08  113k video only         DASH video, mp4_dash
1727177101064097v-1 mp4 216x384    │ ≈  1.20MiB  113k https │ vp09.00.20.08  113k video only         DASH video, mp4_dash
652736553571576v-0  mp4 288x512    │ ~  1.66MiB  156k https │ av01.0.00M.08  156k video only         DASH video, mp4_dash
652736553571576v-1  mp4 288x512    │ ≈  1.66MiB  156k https │ av01.0.00M.08  156k video only         DASH video, mp4_dash
1754038845011986v-0 mp4 288x512    │ ~  1.61MiB  151k https │ vp09.00.21.08  151k video only         DASH video, mp4_dash
1754038845011986v-1 mp4 288x512    │ ≈  1.61MiB  151k https │ vp09.00.21.08  151k video only         DASH video, mp4_dash
806393731144222v-0  mp4 360x640    │ ~  2.41MiB  227k https │ av01.0.01M.08  227k video only         DASH video, mp4_dash
806393731144222v-1  mp4 360x640    │ ≈  2.41MiB  227k https │ av01.0.01M.08  227k video only         DASH video, mp4_dash
945658283213245v-0  mp4 360x640    │ ~  2.37MiB  223k https │ vp09.00.21.08  223k video only         DASH video, mp4_dash
945658283213245v-1  mp4 360x640    │ ≈  2.37MiB  223k https │ vp09.00.21.08  223k video only         DASH video, mp4_dash
640484371378632v-0  mp4 540x960    │ ~  3.72MiB  351k https │ av01.0.04M.08  351k video only         DASH video, mp4_dash
640484371378632v-1  mp4 540x960    │ ≈  3.72MiB  351k https │ av01.0.04M.08  351k video only         DASH video, mp4_dash
1337370123866935v-0 mp4 540x960    │ ~  4.32MiB  407k https │ vp09.00.30.08  407k video only         DASH video, mp4_dash
1337370123866935v-1 mp4 540x960    │ ≈  4.32MiB  407k https │ vp09.00.30.08  407k video only         DASH video, mp4_dash
1246187732768276v-0 mp4 720x1280   │ ~  9.17MiB  864k https │ av01.0.05M.08  864k video only         DASH video, mp4_dash
1246187732768276v-1 mp4 720x1280   │ ≈  9.17MiB  864k https │ av01.0.05M.08  864k video only         DASH video, mp4_dash
590628689648669v-0  mp4 720x1280   │ ~  9.30MiB  876k https │ vp09.00.31.08  876k video only         DASH video, mp4_dash
590628689648669v-1  mp4 720x1280   │ ≈  9.30MiB  876k https │ vp09.00.31.08  876k video only         DASH video, mp4_dash
655904969933390v-0  mp4 1080x1920  │ ~ 15.70MiB 1480k https │ av01.0.08M.08 1480k video only         DASH video, mp4_dash
801218838197353v-0  mp4 1080x1920  │ ~ 33.38MiB 3147k https │ av01.0.08M.08 3147k video only         DASH video, mp4_dash
655904969933390v-1  mp4 1080x1920  │ ≈ 15.70MiB 1480k https │ av01.0.08M.08 1480k video only         DASH video, mp4_dash
801218838197353v-1  mp4 1080x1920  │ ≈ 33.38MiB 3147k https │ av01.0.08M.08 3147k video only         DASH video, mp4_dash
823649719343300v-0  mp4 1080x1920  │ ~ 18.70MiB 1762k https │ vp09.00.40.08 1762k video only         DASH video, mp4_dash
224011970590419v-0  mp4 1080x1920  │ ~ 35.60MiB 3355k https │ vp09.00.40.08 3355k video only         DASH video, mp4_dash
823649719343300v-1  mp4 1080x1920  │ ≈ 18.70MiB 1762k https │ vp09.00.40.08 1762k video only         DASH video, mp4_dash
224011970590419v-1  mp4 1080x1920  │ ≈ 35.60MiB 3355k https │ vp09.00.40.08 3355k video only         DASH video, mp4_dash
```

**Comment by bashonly** [comment]
#4311

FB extractor is currently compeletely broken
